pkg/utils/color.go

```diff
@@ -2,12 +2,27 @@ package utils
 
 import (
 	"regexp"
+	"sync"
 )
 
+var decoloriseCache = make(map[string]string)
```

Comment:
decolorise is quite expensive to run when you're trying to render a lot of stuff so we're caching it here. It would be nice if we delayed colouring things till the very end but that would require some work